aggregator/src/aggregator.rs

```diff
@@ -3175,21 +3175,20 @@ pub(crate) mod test_util {
 
     pub(crate) const BATCH_AGGREGATION_SHARD_COUNT: u64 = 32;
 
-    pub(crate) fn hpke_config_signing_key() -> EcdsaKeyPair {
-        // ring's EcdsaKeyPair does not implement Clone, so we instead store the serialized key &
-        // parse it each time.
-        //
-        // Generated via `openssl ecparam -name prime256v1 -genkey -noout -out -`. Standard
-        // boilerplate: this is a non-sensitive test key, so it is OK that it is checked into a
-        // public GitHub repository. Given that this key is public, it should not be used for any
-        // sensitive purpose.
-        const TEST_HPKE_SIGNING_KEY_PEM: &str = "-----BEGIN EC PRIVATE KEY-----
+    // Generated via `openssl ecparam -name prime256v1 -genkey -noout -out -`. Standard
+    // boilerplate: this is a non-sensitive test key, so it is OK that it is checked into a
+    // public GitHub repository. Given that this key is public, it should not be used for any
+    // sensitive purpose.
```

Comment:
`///` now that this has been (wisely) promoted to doccomment. It'd also be nice if we had a native Rust function to generate this key instead of documenting an `openssl(1)` invocation but let's not fix that here.

Reply:
Alternatively we can use the standard key specified in [RFC 9500](https://www.rfc-editor.org/rfc/rfc9500.html#name-ecdlp-keys), which well-behaved secrets scanning tools on GitHub should understand as a test key.

Reply:
Using a "standard" test key sounds like a good idea, I've taken that example.